library/options.inc.php

```diff
@@ -2446,6 +2446,8 @@ function display_layout_tabs_data($formtype, $result1, $result2='') {
 					$field_id   = $group_fields['field_id'];
 					$list_id    = $group_fields['list_id'];
 					$currvalue  = '';
+                    $condition_str=get_conditions_str($condition_str,$group_fields);
```

Comment:
Minor comment: we prefer spaces before and after most binary operators, and after commas.  Makes the code more readable.